<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_amira</class>
 <widget class="QMainWindow" name="gestion_amira">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>711</width>
      <height>441</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>crud</string>
     </attribute>
     <widget class="QTableView" name="tableau_sponsors">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLineEdit" name="rech">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="sel_col">
      <property name="geometry">
       <rect>
        <x>591</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Adresse</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Montant</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="export_excel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>651</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Excel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>video</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>mail</string>
     </attribute>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>651</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>44</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>subject</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseBtn">
      <property name="geometry">
       <rect>
        <x>587</x>
        <y>79</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>571</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>attachement here</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>116</y>
        <width>651</width>
        <height>291</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>type your message here ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>recipient</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_pass">
      <property name="geometry">
       <rect>
        <x>522</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>your mail password</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>651</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>menu principal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd_sponsor"/>
   <addaction name="actiondelete_sponsor"/>
   <addaction name="separator"/>
   <addaction name="actionstats"/>
  </widget>
  <action name="actionadd_sponsor">
   <property name="text">
    <string>add_sponsor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actiondelete_sponsor">
   <property name="text">
    <string>delete_sponsor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionstats">
   <property name="text">
    <string>stats</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
